bfcfg

```diff
@@ -48,54 +48,130 @@ log_msg()
   echo "$@" >> ${log_file}
 }
 
+mfg_lock()
+{
+  log_msg "mfg: lock the partition"
+  echo 1 > /sys/bus/platform/drivers/mlx-bootctl/mfg_lock 2>>${log_file}
+}
+
 mfg_cfg()
 {
   local tmp_file=/tmp/.bfcfg-mfg-data
-  local opn_str_sysfs=/sys/bus/platform/drivers/mlx-bootctl/opn_str
+  local opn_sysfs=/sys/bus/platform/drivers/mlx-bootctl/opn
+  local sku_sysfs=/sys/bus/platform/drivers/mlx-bootctl/sku
+  local modl_sysfs=/sys/bus/platform/drivers/mlx-bootctl/modl
```

Comment:
Any reason why you didn't want to fully spell out "model".
To me, seems more readable if it's spelled like "model".

Reply:
"MODL" is how the VPD field for Model is named, so the tools which SPE uses also follows MODL